src/app/BUILD.bazel

```diff
@@ -10,26 +10,31 @@ NG_FACTORY_ADDED_IMPORTS = [
     "@npm//@angular/forms",
 ]
 
+# This list is auto-updated by /tools/generator
```

Comment:
these ten things are fixed, not generated, so why bother changing this?

Reply:
I wanted to have a setup where you can run `yarn generate` to scaffold all features. And if you decide to go back to the initial state, `yarn generate:clean` will delete and remove all reference to the ten features.

Currently, running `yarn generate:clean` only deletes the folders/files but doesn't update the Bazel and Angular configurations. The example app breaks because we are referencing deleted files.

This only was done for convenience and allows more flexibility IMHO.

Reply:
I'd rather have less maintenance than flexibility. Please move the new features to a different PR.
In particular, I don't want the generator to be concerned with touching existing files, just stamping out new ones.